### DISCUSSION TITLE
anonymous read-only access, login for write

### DISCUSSION THREAD

**Discussion by andrewleech** [filebrowser/filebrowser#1392] [Ideas]
**Is your feature request related to a problem? Please describe.**
I'm looking to replace an existing basic http file listing where anyone can go straight there and download listed files.

**Describe alternatives you've considered**
The share folder feature is close to what I want, it shows the shared folder to anyone with the link, with a separate button to allow login. 
However this doesn't appear to be available for the root folder, and it has the custom url to get to it.

**Describe the solution you'd like**
Ideally, the root url for the server could go straight to this shared folder read-only interface, rather than directly to the login.

**Additional context**
I'd also like to be able to disable the folder download, removing the qr code and big download button on the shared folder interface and just show the file browser, but that's probably a separate discussion.

**Comment by AetherCollective** [comment]
I'd also like to see this
